Quality_declaration_fastcdr.md

```diff
@@ -0,0 +1,120 @@
+# **External Dependency Quality declaration** `fast-cdr` 
+
+This document is a declaration of software quality for the FastCDR external dependency, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).
+
+The external dependency `fast-cdr` claims to be in the Quality Level 4 category.
```

Comment:
Fast-CDR isn't exactly claiming this. I'd say something like, "This QD claims that the external dependency fast-cdr is ..."